Updated documentation for running ESM1.5 and OM3 to reflect updates to best practices for using Payu (#1252)
ACCESS-Hive/access-hive.github.io: docs/models/run_a_model/run_access-esm.md

Reviewer: Deleting troubleshooting explanations about `payu setup` as already included in the `setup` step.

!!! admonition warning
    Files on the `/scratch` drive, such as the _laboratory_ directory, might get deleted if not accessed for several days and the `/scratch` drive is limited in space. For these reasons, all model runs which are to be kept should be moved to `/g/data/` by enabling the _sync_ step in _payu_. To know more refer to [Syncing output data](#syncing-output-data).

### Setup configuration
Before running a newly cloned configuration for the first time, run the following command in the control directory:

    payu setup

This command will: 
  

>   - create the _laboratory_ and `work` directories based on the experiment configuration

Reviewer: ```suggestion
  - create _laboratory_, `work` and `archive` directories based on the experiment configuration,
```

>   - create the _laboratory_ and `work` directories based on the experiment configuration

Reviewer: Can't remember why we use italics for _laboratory_? `work` and `archive` are formatted as code blocks.

>   - create the _laboratory_ and `work` directories based on the experiment configuration

Reviewer 2: Good question. I am not sure either. My guess is laboratory name is something like `/home/189/$USER/model`, so it changes as different models. However, `work` and `archive` are fixed names.

>   - generate manifests

Reviewer: ```suggestion
  - create or update manifests,
```

  - report useful information to the user, such as the location of the _laboratory_ where the `work` and `archive` directories are located

> 

Reviewer: Thanks @Qian-HuiChen! Added your suggestion [above](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1252/changes#r3793037002).

!!! admonition warning
    Files on the `/scratch` drive, such as the _laboratory_ directory, might get deleted if not accessed for several days and the `/scratch` drive is limited in space. For these reasons, all model runs which are to be kept should be moved to `/g/data/` by enabling the _sync_ step in _payu_. To know more refer to [Syncing output data](#syncing-output-data).

### Setup configuration
Before running a newly cloned configuration for the first time, run the following command in the control directory:

    payu setup

This command will: 
  
  - create the _laboratory_ and `work` directories based on the experiment configuration
  - generate manifests

>   - report useful information to the user, such as the location of the _laboratory_ where the `work` and `archive` directories are located

Reviewer: ```suggestion
- load any modules specified in the `config.yaml` file
- report useful information, such as the location of the _laboratory_, including the `work` and `archive` directories

```

    <terminal-line linedelay=0 class="keep-blanks">&lt;job-ID&gt;.gadi-pbs      &lt;other-job-name&gt; &lt;\$USER&gt;           &lt;time&gt;   R normal-exec</terminal-line>
</terminal-window>

To monitor your job's status continuously without repeatedly executing `qstat`, use the following command:
```

> watch -n 30 payu status

Reviewer: Thanks for clarifying @Qian-HuiChen! Changes suggested according to [below](https://github.com/ACCESS-NRI/ACCESS-Hive-Docs/pull/1252/changes#r3754279556) comment:

```suggestion
payu status
```
